gemini-extension.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "pinecone-mcp",
+  "version": "1.0.0",
+    "mcpServers": {
+      "pinecone": {
+        "command": "npx",
+        "args": [
+          "-y", "@pinecone-database/mcp"
+        ]
+      }
+    }
+}
```

nit consistent indent
```suggestion
{
  "name": "pinecone-mcp",
  "version": "1.0.0",
  "mcpServers": {
    "pinecone": {
      "command": "npx",
      "args": [
        "-y", "@pinecone-database/mcp"
      ]
    }
  }
}
```